docs/en_quick_starting_guide.txt

```diff
@@ -151,7 +151,7 @@ When upgrading from OpenWrt, in order to connect via ssh/scp you will need to ha
 Check out link:/docs/en_connecting_nodes.html[Connecting to your own node] for help on this.
 ===========
 
-Now connect to the console interface using +ssh+, if LibreMesh is already running with +ssh root@thisnode.info+ or with +ssh root@ROUTERIPADDRESS+ if OpenWrt or LEDE are running.
+Now connect to the console interface using +ssh+, if LibreMesh is already running with +ssh \root@thisnode.info+ or with +ssh root@ROUTERIPADDRESS+ if OpenWrt or LEDE are running.
```

Comment:
I based this change off of the explanation at http://mrhaki.blogspot.com/2017/10/awesome-asciidoctor-prevent.html.